website/static/js/filepage/index.js

```diff
@@ -157,6 +158,72 @@ var FileViewPage = {
         }, 1000);
 
         self.mfrIframeParent = $('#mfrIframeParent');
+
+        $.fn.editable.defaults.mode = 'inline';
```

Comment:
Why is this set globally? It might be better to pass this to `.editable` instead to make sure this doesn't affect other editable inputs.